arrow-data-source/standard/src/main/scala/com/intel/oap/spark/sql/ArrowWriteExtension.scala

```diff
@@ -71,8 +70,9 @@ object ArrowWriteExtension {
       case rc @ DataWritingCommandExec(cmd, ColumnarToRowExec(child)) =>
         cmd match {
           case command: InsertIntoHadoopFsRelationCommand =>
-            if (command.fileFormat
-                  .isInstanceOf[ArrowFileFormat]) {
+            // TODO: support writing parquet fileformat
+            if (command.fileFormat.isInstanceOf[ArrowFileFormat] &&
+                !command.fileFormat.isInstanceOf[ParquetFileFormat]) {
```

Comment:
A naive question here: Didn't `command.fileFormat.isInstanceOf[ArrowFileFormat]` incorporate `!command.fileFormat.isInstanceOf[ParquetFileFormat]`?

Reply:
We have overwrited the `ParquetFileFormat`, it is now a subclass of `ArrowFileFormat`. Thus, if fileformat is `ParquetFileFormat`, it will pass `command.fileFormat.isInstanceOf[ArrowFileFormat]`.